src/InteractiveBridge.ts

```diff
@@ -72,6 +72,38 @@ export class InteractiveBridge extends LiveryBridge {
     });
   }
 
+  /**
+   * Returns from the LiveryPlayer's context an object of key-value parameters
+   * where the keys start 'livery_' and the values are strings. Used for retrieving
+   * livery specific query parameters from the web player's context. Delegates to
+   * callbacks for fulfillment in a native context.
+   */
+  getLiveryParams() {
+    return this.sendCommand('getLiveryParams').then((value) => {
+      if (typeof value !== 'object') {
+        throw new Error(
+          `getLiveryParams value type: ${typeof value}, should be: object`,
+        );
+      }
+      if (value === null) {
+        throw new Error(`getLiveryParams value type: null, should be: object`);
+      }
+      for (const [k, v] of Object.entries(value)) {
```

Comment:
We normally don't use single letter variables. How about using paramKey and paramValue here?

Reply:
Ah I tend to use as a general code style rule, and this comes from Robert C Martin, the shorter the lifespan of a variable the shorter the name can be. I have found benefits to that beyond terseness: variables with very short lifespans often have more abstract concepts; variables with short lifespans are often used several times in that period; when you see a variable with a short name, you know it is short lived, probably just a temporary convenience, possibly very abstract -- nothing to do with the module interface.

I just thought you'd be interested where this came from. I'll change to paramValue (k is no longer needed)

Reply:
It's a quite subjective topic. For example with golang they explicitly do like you to use 1 letter variables :-P. 
I agree we should pick a strategy and let everyone use that as much as possible within a single code base.

Do we have this rule (don't use super short variable names) written down somewhere?

Reply:
I came into Javascript development from a Java background with long names (before that time I had also used single letter names for gaming and functional programming). I have since then adjusted to use more terse but still always descriptive (i.e: at least a single word) names in line with general conventions.

I do also follow a similar philosophy of using shorter names in more specific context. I.e: not needlessly repeating information that can already be derived from the context.

We could add a line about this to our CONVENTIONS.md file; I don't see this there yet. But we do already mention that our lint rules are based on Airbnb's style guide and link to that. In this case see: https://github.com/airbnb/javascript#naming-conventions

Looking at this again now I see that perhaps we could enforce these conventions to some extent instead of just documenting them: https://github.com/airbnb/javascript#naming-conventions

That could be interesting as I generally prefer to have automatic convention enforcement (when possible including auto fixing) instead of lengthy documents to memorize. But I suppose with naming conventions like these we could not really automatically determine the descriptiveness. E.g: we could require at least say 3 characters in names, but that would make working with perfectly understandable short names like x and y with respect to coordinates illegal. The only thing this linter seems to be good at is enforcing consistent use of case and underscore.